src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -61,7 +61,16 @@ int CondTraverseToString(const OpBase *ctx, char *buff, uint buff_len) {
 	offset += snprintf(buff + offset, buff_len - offset, "%s | ", op->op.name);
 	offset += QGNode_ToString(op->ae->src_node, buff + offset, buff_len - offset);
 	if(op->ae->edge) {
-		if(op->ae->operands[0].transpose) {
+		// Determine whether thise edge should be printed left-to-right or right-to-left.
+		bool transpose = false;
+		for(uint i = 0; i < op->ae->operand_count; i ++) {
```

Comment:
This loop need some explanation, if an edge is referenced then the algebraic expression...